packages/utilities/gel-foundations/scripts.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable global-require */
+module.exports = {
+  news: require('./dist/scripts/news').default,
+  arabic: require('./dist/scripts/arabic').default,
+  burmese: require('./dist/scripts/burmese').default,
+  cjkThai: require('./dist/scripts/cjk_thai').default,
+  hindiNepaliBengali: require('./dist/scripts/hindi_nepali_bengali').default,
```

Comment:
Since we're defining scripts, not services, we should name these accordingly.

Hindi & Nepali use the `Devanagari` script, so `devanagari` should be the script name here.  (https://en.wikipedia.org/wiki/Devanagari)

Bengali uses the `Bengali-Assamese` script and should be named as a separate file.  https://en.wikipedia.org/wiki/Bengali%E2%80%93Assamese_script

We shouldn't just group different scripts together for convenience, but have them be distinct files.

Reply:
@sareh Applied the new scripts group names.